Dockerfile

```diff
@@ -15,6 +15,10 @@ ARG PLATFORM="linux/amd64"
 ARG R_VERS="4.2.3"
 FROM --platform=$PLATFORM rocker/r-ver:$R_VERS
 
+LABEL org.opencontainers.image.source=https://github.com/RMI-PACTA/workflow.transition.monitor
+LABEL org.opencontainers.image.description="Docker image to drive the Transition Monitor backend"
+LABEL org.opencontainers.image.licenses=MIT
```

Comment:
NIT: use quotes for consistency?

Reply:
Honestly, I'm not 100% sure what that will do. These lines were copied then edited directly from GitHub's directions [here](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry#labelling-container-images), so I don't know if certain labels require quotes and others do not, therefore I am hesitant to mess with it. Note that the code from Rocker that I posted is in a different syntax (JSON).